source/_integrations/edl21.markdown

```diff
@@ -50,6 +51,20 @@ serial_port:
   type: string
 {% endconfiguration %}
 
+## InF Mode
+
+To enable InF mode there are different steps needed based on the meter type but most commonly you have to enter the PIN you received from your grid operator. Once you have it, enter it to the meter and switch to the InF menu where you can switch from InF=Off to InF=On. 
+Entering this can be done using a flashlight or (if available) via physical button on the meter.
```

Comment:
```suggestion
To enable InF mode there are different steps needed based on the meter type but most commonly you have to enter the PIN you received from your grid operator. Once you have it, enter it into the meter and switch to the InF menu where you can switch from InF=Off to InF=On. 
Entering this can be done using a flashlight or (if available) via the physical button on the meter.
```